src/ApiTests/ApiApproval.cs

```diff
@@ -1,22 +1,17 @@
-﻿using System.IO;
-using System.Reflection;
-using System.Runtime.CompilerServices;
-using ApprovalTests;
-using ApprovalTests.Reporters;
+﻿using System.Runtime.CompilerServices;
+using NServiceBus.Persistence.ServiceFabric;
 using NUnit.Framework;
+using Particular.Approvals;
 using PublicApiGenerator;
 
 [TestFixture]
 public class APIApprovals
 {
     [Test]
     [MethodImpl(MethodImplOptions.NoInlining)]
-    [UseReporter(typeof(DiffReporter), typeof(AllFailingTestsClipboardReporter))]
     public void ApproveServiceFabricPersistence()
     {
-        var combine = Path.Combine(TestContext.CurrentContext.TestDirectory, "NServiceBus.Persistence.ServiceFabric.dll");
-        var assembly = Assembly.LoadFile(combine);
-        var publicApi = ApiGenerator.GeneratePublicApi(assembly);
-        Approvals.Verify(publicApi);
+        var publicApi = ApiGenerator.GeneratePublicApi(typeof(ServiceFabricPersistence).Assembly);
```

Comment:
When switching to Particular.Approvals, you can get get rid of the `[MethodImpl(MethodImplOptions.NoInlining)]` bit.